src/seedu/addressbook/parser/Parser.java

```diff
@@ -59,7 +63,7 @@
  */
 public class Parser {
 
-    public static final Pattern PERSON_INDEX_ARGS_FORMAT = Pattern.compile("(?<targetIndex>.+)");
+    public static final Pattern GENERAL_ARGS_FORMAT = Pattern.compile("(?<keywords>.+)");
 
```

Comment:
Huh. Why did rename it to be General_Args? I thought KEYWORDS_ARGS describes the pattern of General_Args?

Reply:
actually why keywords args format was added if it checks for exactly the same thing as this line?

Reply:
oops mb. changed this back